packages/ag-charts-website/src/content/docs/api-create-update/index.mdoc

```diff
@@ -8,25 +8,33 @@ Learn about creating and updating charts in more detail.
 ## Creating and Updating Charts
 
 {% if isFramework("javascript") %}
-`AgCharts` exposes `create()` and `update()` static methods to perform chart initialisation and update based upon
-the `AgChartOptions` configuration structure.
+`AgCharts` exposes `create()` static methods to perform chart initialisation, and the resulting `AgChartInstance` has methods
+such as `AgChartInstance.update()` to allow updating configuration. The `AgChartOptions` type defines the configuration structure.
 
 Mutations to the previously used options object are not automatically picked up by the chart implementation,
-`AgCharts.update()` should be called for changes to be applied.
+`AgChartInstance.update()` should be called for changes to be applied.
 
 {% note %}
-We expect the options supplied to `AgCharts.update()` to be the full configuration state to update
-to, not a partial configuration. Use `AgCharts.updateDelta()` to apply partial updates.
+We expect the options supplied to `AgChartInstance.update()` to be the full configuration state to update
+to, not a partial configuration. Use `AgChartInstance.updateDelta()` to apply partial updates.
```

Comment:
This may cause some people to expect to be able to call `AgChartInstance.update(chart)` instead of `chart.update()`?

Reply:
Maybe, of the two style options `chart.update()` and `AgChartInstance.update()` I'm leaning towards the later as it has a stronger binding to actual types, over the convention of `chart`.

We talk about getting an `AgChartInstance` from the `AgCharts.create()` API, and the `AgChartsInstance` API section is just before this too, so I'm hoping it's not much of a leap 🤞